secretless-broker.rb

```diff
@@ -5,20 +5,28 @@
 class SecretlessBroker < Formula
   desc "Secures your apps by making them Secretless"
   homepage "https://secretless.io"
-  version "1.7.2"
-  bottle :unneeded
+  version "1.7.9"
 
-  if OS.mac?
-    url "https://github.com/cyberark/secretless-broker/releases/download/v1.7.2/secretless-broker_1.7.2_darwin_amd64.tar.gz"
-    sha256 "9b86ae7824b6a5025b2eb85f963d1c7b991dea6b387f68540429034d7f448182"
-  end
-  if OS.linux? && Hardware::CPU.intel?
-    url "https://github.com/cyberark/secretless-broker/releases/download/v1.7.2/secretless-broker_1.7.2_linux_amd64.tar.gz"
-    sha256 "1c7857d7995b86089abf73cfad3208553fe16b94d099f5b746e01ba9123f9364"
+  on_macos do
+    if Hardware::CPU.intel?
```

Comment:
We should support mac arm also.

Reply:
Do we need to make changes to the goreleaser process in secretless-broker? This file is generated automatically.

Reply:
Yes. In summon it was added with [#217](https://github.com/cyberark/summon/pull/217)